### DISCUSSION TITLE
Дальнейшее развитие приложения

### DISCUSSION THREAD

**Discussion by sukrekus** [ferscalone/multifunctional-calculator#9] [Q&A] [answered]
Планируешь ли дальше развивать этот проект и, если да, то когда?

**Comment by ferscalone** [comment, marked as answer]
Да, это приложение кажется уж очень недоработанным, поэтому будут изменения в дизайне, привязка элементов к layouts и прочее.
Сейчас доделывается и где-то переделывается блок алгебры, затем уже можно будет приступать к реализации других блоков.
На счёт сроков сказать точно не могу :)
